store-rx2/src/main/kotlin/com/dropbox/store/rx2/RxStore.kt

```diff
@@ -0,0 +1,96 @@
+package com.dropbox.store.rx2
+
+import com.dropbox.android.external.store4.BuilderImpl
+import com.dropbox.android.external.store4.BuilderWithSourceOfTruth
+import com.dropbox.android.external.store4.ExperimentalStoreApi
+import com.dropbox.android.external.store4.Store
+import com.dropbox.android.external.store4.StoreBuilder
+import com.dropbox.android.external.store4.StoreRequest
+import com.dropbox.android.external.store4.StoreResponse
+import io.reactivex.Flowable
+import io.reactivex.Maybe
+import io.reactivex.Single
+import kotlinx.coroutines.reactive.asFlow
+import kotlinx.coroutines.rx2.asFlowable
+import kotlinx.coroutines.rx2.await
+import kotlinx.coroutines.rx2.rxSingle
+
+/**
+ * Return a flow for the given key
+ * @param request - see [StoreRequest] for configurations
+ */
+fun <Key : Any, Output : Any> Store<Key, Output>.observe(request: StoreRequest<Key>): Flowable<StoreResponse<Output>> =
```

Comment:
i'm not sure about the `observe` name, would this make people assume this would return observable rather than a flowable ?

Reply:
I went back and forth on this, I never used a flowable myself in the 5 years of using rxjava. Maybe we should just yolo and return an observable with backpressure latest?

Reply:
leaving as flow for now, will get user feedback internally and flip if necessary